aospy/test/test_roundtrip.py

```diff
@@ -184,5 +198,89 @@ def test_monthly_ts(self):
+                                 'date_range': ('1981-01-01',
+                                                '2010-12-31'),
+                                 'intvl_in': 'monthly',
+                                 'dtype_in_time': 'ts',
+                                 'level': False}
+
+
+@unittest.skipIf(not model_files_exist(hiram), skip_message)
+class TestHiRAM(TestAM2):
+    def setUp(self):
+        self.olr_test_params = {'proj': aospy_test,
+                                'model': hiram,
+                                'run': test_hiram,
+                                'var': olr,
+                                'date_range': ('1979-01-01', '1995-12-31'),
+                                'intvl_in': 'monthly',
+                                'dtype_in_time': 'ts',
+                                'dtype_in_vert': 'pressure',
+                                'dtype_out_vert': False,
+                                'level': False}
+        self.temp_test_params = {'proj': aospy_test,
+                                 'model': hiram,
+                                 'run': test_hiram,
+                                 'var': temp,
+                                 'date_range': ('1979-01-01',
+                                                '1995-12-31'),
+                                 'intvl_in': 'monthly',
+                                 'dtype_in_time': 'ts',
+                                 'level': False}
+
+    @unittest.skip(('could not immediately find a case with temperature'
+                    ' output on sigma levels in CMIP5 archive'))
+    def test_vert_int_sigma(self):
+        pass
+
+
+# cesm1_cam5 has no grid files so just use hiram's to test to skip
+@unittest.skipIf(not model_files_exist(hiram), skip_message)
+class TestCMIP5(TestAM2):
+    def setUp(self):
+        self.olr_test_params = {'proj': aospy_test,
+                                'model': cesm1_cam5,
+                                'run': test_amip,
+                                'var': olr,
+                                'date_range': ('1979-01-01', '2008-12-31'),
+                                'intvl_in': 'monthly',
+                                'dtype_in_time': 'ts',
+                                'dtype_in_vert': 'pressure',
+                                'dtype_out_vert': False,
+                                'level': False}
+        self.temp_test_params = {'proj': aospy_test,
+                                 'model': cesm1_cam5,
+                                 'run': test_amip,
+                                 'var': temp,
+                                 'date_range': ('1979-01-01',
+                                                '2008-12-31'),
+                                 'intvl_in': 'monthly',
+                                 'dtype_in_time': 'ts',
+                                 'level': False}
+
+    @unittest.skip(('could not immediately find a case with temperature'
```

Comment:
I think the CMIP5 data is all interpolated to regular pressure levels, so we'll just have to omit this test.